API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
+description | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+url | UrlField  
+difficulty | CharField (choices)  
+duration | CharField (choices)  
+**Skills** | *:*  
+**WebLitSkills** | *:*  
+**Topic** | 1:*  
+**LessonPlan** | *:*  
+**locale** | CharField (choices) (pycountry)  
+translation_of | ‘self’  
+
+
+
+<pre>Skills</pre>
```

Comment:
Are "Skills" the four 21st Century Skills that have been identified (Problem Solving, Collaboration, Creativity, and Communication) and are "WebLitSkills" the 14 skills around the outer ring of the web lit map (Design, Revise, Synthesize, Navigate, Protect, etc.)?